src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -4690,6 +4698,24 @@ def get_node_resource_group(self) -> Union[str, None]:
         # this parameter does not need validation
         return node_resource_group
 
+    def get_k8s_support_plan(self) -> Union[str, None]:
+        """Obtain the value of kubernetes_support_plan.
+
+        :return: string or None
+        """
+        # default to None
+        support_plan = None
+        # try to read the property value corresponding to the parameter from the `mc` object
+        if self.mc and self.mc.support_plan is not None:
```

Comment:
```suggestion
        if self.mc and hasattr(self.mc, "support_plan") and self.mc.support_plan is not None:
```

should fix hybrid test